unified-installer/astra-unified-installer.sh

```diff
@@ -413,15 +432,15 @@ components_include_acp() {
 }
 
 get_trident_namespace() {
-    echo "${_EXISTING_TRIDENT_NAMESPACE:-"${NAMESPACE:-"${__DEFAULT_TRIDENT_NAMESPACE}"}"}"
+    echo "${_EXISTING_TRIDENT_NAMESPACE:-"${TRIDENT_NAMESPACE:-"${NAMESPACE:-"$__DEFAULT_TRIDENT_NAMESPACE"}"}"}"
 }
 
 get_connector_operator_namespace() {
-    echo "${NAMESPACE:-"${__DEFAULT_CONNECTOR_OPERATOR_NAMESPACE}"}"
+    get_connector_namespace
```

Comment:
`__DEFAULT_CONNECTOR_OPERATOR_NAMESPACE` didn't actually exist, and, as far as the script is concerned, this needs to always be in the connector namespace.